test/periphery/CardPaymentProcessorUpgradeable.test.ts

```diff
@@ -0,0 +1,1415 @@
+import { ethers, upgrades } from "hardhat";
+import { expect } from "chai";
+import { BigNumber, Contract, ContractFactory } from "ethers";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/dist/src/signer-with-address";
+import { proveTx } from "../../test-utils/eth";
+import { createBytesString } from "../../test-utils/misc";
+
+const BYTES16_LENGTH: number = 16;
+const BYTES32_LENGTH: number = 32;
+const ZERO_BYTES16_STRING: string = createBytesString('00', BYTES16_LENGTH);
```

Comment:
Codacy has a fix for the issue: [Replace `'00'` with `"00"`](https://app.codacy.com/gh/cloudwalk/brlc-token/pullRequest?prid=9781596)
```suggestion
const ZERO_BYTES16_STRING: string = createBytesString("00", BYTES16_LENGTH);
```